binding.gyp

```diff
@@ -7,7 +7,10 @@
                 'src/HDFileSystem.cc',
                 'src/HDFile.cc'
             ],
-            'cflags' : ['-Wall', '-Wextra', '-Wno-unused-parameter'],
+            'xcode_settings': {
+                'OTHER_CFLAGS': ['-Wno-unused-parameter', '-Wno-unused-result']
```

Comment:
will it build on a mac now?

Reply:
No I didn't actually fix that issue although I also didn't really try. I'll take a quick look and see if I can add some flags to get it running. For now I was testing in a container.

Reply:
I was able to get it fixed and building on my mac.